core/controllers/question_editor_test.py

```diff
@@ -229,6 +291,14 @@ def setUp(self):
             self.question_id, self.skill_id)
 
     def test_get(self):
+        with self.swap(constants, 'ENABLE_NEW_STRUCTURE_EDITORS', False):
```

Comment:
- The function names in the test don't indicate the exact behaviour which is being tested. I see that it is throughout the file apart from your changes. Can you please try renaming all the functions to descriptive names which explain the purpose of test.

- Try testing one behaviour per test. If you are unable to name a test which indicates what behaviour it is testing, split it into multiple tests and ensure that one behaviour is tested in a single function.

For instance, you can split this test as follows:

- `test_get_with_new_structures_disabled_returns_404_status`
- `test_get_with_new_structures_enabled_does_not_allow_access_to_non_admin`
- `test_get_with_new_structures_enabled_allows_access_to_admin`
- `test_get_with_new_structures_enabled_allows_access_to_topic_manager`
- `test_get_with_new_structures_enabled_allows_access_to_question_creator`

Reply:
I have changed all the tests in the class in format stated.